pallet-logion-loc/src/lib.rs

```diff
@@ -732,17 +741,10 @@ pub mod pallet {
         pub fn add_metadata(
             origin: OriginFor<T>,
             #[pallet::compact] loc_id: T::LocId,
-            item: MetadataItemParams<T::AccountId, T::EthereumAddress>
+            item: MetadataItemParams<T::AccountId, T::EthereumAddress, <T as pallet::Config>::Hash>
         ) -> DispatchResultWithPostInfo {
             let who = ensure_signed(origin)?;
 
-            if item.name.len() > T::MaxMetadataItemNameSize::get() {
-                Err(Error::<T>::MetadataItemInvalid)?
-            }
-            if item.value.len() > T::MaxMetadataItemValueSize::get() {
-                Err(Error::<T>::MetadataItemInvalid)?
-            }
```

Comment:
All those `Max...Size` parameters can be removed from config I guess.

Reply:
Indeed, I forgot to remove them.